src/main/kotlin/com/terraformation/backend/accelerator/db/CohortStore.kt

```diff
@@ -57,6 +64,8 @@ class CohortStore(
 
     cohortsDao.insert(row)
 
+    assignCohortModules(row.toModel())
+
```

Comment:
Could save `row.toModel()` to a variable and use it both here and in the `return` statement.